<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>verapp::ProjectConfigPage</class>
 <widget class="QWidget" name="verapp::ProjectConfigPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>917</width>
    <height>848</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="AdditionalChecksGroupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Built-in Rules</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="tabPosition">
         <enum>QTabWidget::North</enum>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tabF">
         <attribute name="title">
          <string>F</string>
         </attribute>
        </widget>
        <widget class="QWidget" name="tabL">
         <attribute name="title">
          <string>L</string>
         </attribute>
        </widget>
        <widget class="QWidget" name="tabT1">
         <attribute name="title">
          <string>T1</string>
         </attribute>
        </widget>
        <widget class="QWidget" name="tabT2">
         <attribute name="title">
          <string>T2</string>
         </attribute>
        </widget>
        <widget class="QWidget" name="tabT3">
         <attribute name="title">
          <string>T3</string>
         </attribute>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>File Filter</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="kcfg_fileFilter">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>&amp;Extra parameters:</string>
       </property>
       <property name="buddy">
        <cstring>kcfg_extraParameters</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="kcfg_extraParameters">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Defines additional parameters for vera++ (see documentation).&lt;/p&gt;&lt;p&gt;You can use the following placeholders:&lt;/p&gt;&lt;p&gt;%p - Gets replaced by the URL of the project's root directory.&lt;/p&gt;&lt;p&gt;%b - Gets replaced by the URL of the project's build directory.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Command line</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="commandLineFilterLabel">
          <property name="text">
           <string>&amp;Filter:</string>
          </property>
          <property name="buddy">
           <cstring>commandLineFilter</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="commandLineFilter">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="commandLineBreaks">
          <property name="text">
           <string>Break lines</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTextEdit" name="commandLine">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="acceptRichText">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>kcfg_fileFilter</tabstop>
  <tabstop>kcfg_extraParameters</tabstop>
  <tabstop>commandLineFilter</tabstop>
  <tabstop>commandLineBreaks</tabstop>
  <tabstop>commandLine</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>commandLineBreaks</sender>
   <signal>toggled(bool)</signal>
   <receiver>commandLineFilter</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>901</x>
     <y>513</y>
    </hint>
    <hint type="destinationlabel">
     <x>772</x>
     <y>518</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>commandLineBreaks</sender>
   <signal>toggled(bool)</signal>
   <receiver>commandLineFilterLabel</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>901</x>
     <y>513</y>
    </hint>
    <hint type="destinationlabel">
     <x>54</x>
     <y>518</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
